.github/actions/get-branch-name/action.yml

```diff
@@ -0,0 +1,10 @@
+name: "Get branch name"
```

Comment:
if you move this and get-commit-message in a separated PR we can merge it in advance so you can have a working reference (`@master`?)

Reply:
I created https://github.com/Alfresco/alfresco-build-tools/pull/107